src/views/GRTSystemIntakeReview/index.tsx

```diff
@@ -1,14 +1,7 @@
 import React, { useEffect } from 'react';
-import { withAuth } from '@okta/okta-react';
 import { useDispatch, useSelector } from 'react-redux';
 import { RouteComponentProps } from 'react-router-dom';
-import { DateTime } from 'luxon';
-import {
-  DescriptionList,
-  DescriptionTerm,
-  DescriptionDefinition
-} from 'components/shared/DescriptionGroup';
-import convertBoolToYesNo from 'utils/convertBoolToYesNo';
+import { SystemIntakeReview } from 'components/SystemIntakeReview';
```

Comment:
Ideally we'd want this to  be a default import. It looks like in `src/components/SystemIntakeReview/index.tsx`, SystemIntakeReview` is exported both named and defaultly.